packages/python-Wappalyzer/PKGBUILD

```
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-Wappalyzer
```

```suggestion
pkgname=python-python-wappalyzer
```

never put upper case in pkgname also the convention is `python-<pythonpkgname>` even if that makes  `python-python-something`